### ISSUE TITLE
Issue while connecting to iOS device via USB

### ISSUE THREAD

**Issue by anuragrao04** [dev47apps/droidcam#183]
**Describe the bug**
When I try to connect to my iOS device via USB, an error is thrown asking me to ensure usbmuxd is installed and enables. I use Arch and usbmuxd is installed via pacman. 

**Expected behavior**
Connectivity to iOS device via USB 

**Additional context**
OS: Arch LInux
Kernel: 5.12
iOS: 14.6
Device: iPhone 7 Plus

**Comment by aramg** [comment]
Did you install the client from the dev47apps website, or through AUR?

**Comment by anuragrao04** [comment]
Through GitHub. I had cloned the repository and installed using the script.

**Comment by aramg** [comment]
Could you please try the pre-built release: https://github.com/dev47apps/droidcam/releases/tag/v1.7.3
There have been reports of things not working 100% with it on arch

**Comment by anuragrao04** [comment]
I'm afraid the issue persists. I went through the readme file once again to see if I missed any libraries. I also ensured usbmuxd was installed. Any help on this would be highly appreciated!

**Comment by aramg** [comment]
OK, can you use `idevice_id` and see if it lists the phone?
https://man.archlinux.org/man/extra/libimobiledevice/idevice_id.1.en

If it does, try using iproxy (part of same package) to create a local proxy and connect like that.
Your version of iproxy might have different usage, but roughly
```
$ iproxy -h
iproxy [LOCAL_TCP_PORT] [DEVICE_TCP_PORT] [UDID]
$ iproxy 1111 4747
$ droidcam-cli localhost 1111

**Comment by anuragrao04** [comment]
Hey! usbmuxd was the problem. I discovered it today morning while rereading the arch wiki on usbmuxd. usbmuxd.service is supposed to start automatically when an idevice is connected. Unfortunately in my case - it didn't. I manually started the service using systemd and the bug was resolved. In order to solve the issue, I wrote a script to manually start the service on startup and it works flawlessly now. Thanks for your time and efforts!
